content/intro-to-storybook/react/en/test.md

```diff
@@ -28,88 +28,44 @@ Visual regression tests are designed to catch changes in appearance. They work b
 
-Then [login to Chromatic](https://www.chromatic.com/start) with your GitHub account (Chromatic only asks for lightweight permissions). Create a project with name "taskbox" and copy your unique `project-token`.
+Let's see how it works by tweaking the background of the `Task` component:
 
-<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
-  <source
-    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
-    type="video/mp4"
+```js
+// src/components/Task.js
+<div className="title">
+  <input
+    type="text"
+    value={title}
+    readOnly={true}
+    placeholder="Input title"
+    style={{ textOverflow: 'ellipsis', background: 'red' }}
   />
-</video>
-
-Run the test command in the command line to setup visual regression tests for Storybook. Don't forget to add your unique project token in place of `<project-token>`.
-
-```bash
-npx chromatic --project-token=<project-token>
-```
-
-<div class="aside">
-If your Storybook has a custom build script you may have to <a href="https://www.chromatic.com/docs/setup#command-options">add options</a> to this command.
 </div>
-
-Once the first test is complete, we have test baselines for each story. In other words, screenshots of each story known to be “good”. Future changes to those stories will be compared to the baselines.
-
-![Chromatic baselines](/intro-to-storybook/chromatic-baselines.png)
-
-## Catch a UI change
-
-Visual regression testing relies on comparing images of the new rendered UI code to the baseline images. If a UI change is caught you get notified. See how it works by tweaking the background of the `Task` component:
-
-![code change](/intro-to-storybook/chromatic-change-to-task-component.png)
+```
 
 This yields a new background color for the item.
 
 ![task background change](/intro-to-storybook/chromatic-task-change.png)
 
-Use the test command from earlier to run another Chromatic test.
-
-```bash
-npx chromatic --project-token=<project-token>
-```
-
-Follow the link to the web UI where you’ll see changes.
+Now once we commit and push the changes to our remote repo, our action will execute and not only update our Storybook with the new change, but also generate a new set of changes that need to reviewed.
 
-![UI changes in Chromatic](/intro-to-storybook/chromatic-catch-changes.png)
+![actions panel](/github/gh-actions-panel.png)
 
-There are a lot of changes! The component hierarchy where `Task` is a child of `TaskList` and `Inbox` means one small tweak snowballs into major regressions. This circumstance is precisely why developers need visual regression testing in addition to other testing methods.
+Once our action finishes it's execution we'll see a screen like the following. Scrolling through it we'll see what was done but also we get a link that will redirect us to our Chromatic dashboard. In there we're presented with a multitude of options and amongst them we can also get a link to our deployed Storybook also.
```

Comment:
```suggestion
Once our action finishes, we'll see the following screen.
```

Reply:
Resolved with the latest commits. Marking it as closed